README.md

```diff
@@ -371,9 +371,9 @@ Carbon focused talks from the community:
 
 -   [The Carbon Language: Road to 0.1](https://chandlerc.blog/slides/2024-ndc-techtown-carbon-road-to-0-dot-1),
     NDC {TechTown}
--   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://chandlerc.blog/slides/2024-cppnorth-design-stories),
+-   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://youtu.be/8SGMy9ENGz8),
```

Comment:
In every other case, we only have the youtube video. I think that if the style is going to change, we should do that for everything instead of a one-off on this PR.

Reply:
Also see #3158 for some history on this.

Reply:
Keeping slides would be a nice accessibility improvement, I think

Reply:
Putting myself in the shoes of someone viewing this site, I would be interested in a link to the slide and would not otherwise know if one existed.

Reply:
Sorry, it sounds like this is turning into a discussion instead of a simple update, and I'm about to go out for a week. I've commented my concerns on #4578 and will close this PR.